lib/EntityGenerator/src/Field/AbstractFieldGenerator.php

```diff
@@ -11,6 +11,7 @@
 use Nette\PhpGenerator\Property;
 use RZ\Roadiz\Contracts\NodeType\NodeTypeFieldInterface;
 use RZ\Roadiz\Contracts\NodeType\SerializableInterface;
+use RZ\Roadiz\CoreBundle\Enum\FieldType;
```

Comment:
```suggestion
```

Warning: you cannot use CoreBundle classes, enum and other types. Only use `NodeTypeFieldInterface` methods